### ISSUE TITLE
Validation Tool topology check - feature class selectioin

### ISSUE THREAD

**Issue by mdhendricks** [usgs/gems-tools-pro#28]
We here at AK DGGS have a number of maps that are multilayered in one DB (~bedrock & surficial).  The GeMS validator results in many topological errors with these databases (overlaps and gaps). This is a known and understandable issue at this point.  In an attempt to work within the current set of tools we tried splitting out the surficial map units into MapUnitSurficial and contacts and faults into ContactsAndFaultsSurficial. See an attached screenshot of our DB schema
![image](https://user-images.githubusercontent.com/108186228/180283639-e75b6678-7652-4de8-9463-ad84647fd279.png)
What is interesting with the tool is it appears that it selects the first (maybe last) feature class it finds with MapUnit and ContactsAndFaults in the name, so in our case it is picking the surficial layers to check topology, not the primary bedrock layers.

It seems like we would want the topology tool to pick explicitly the feature classes with the exact names. See an attached screenshot of the validator tool database that is created that includes only the surficial versions of the two primary FCs
![image](https://user-images.githubusercontent.com/108186228/180284501-23548008-2dea-4270-ba1d-eb353de8b19a.png)

**Comment by ethoms-usgs** [comment]
Yes, though the tool does try to allow for prefixes or suffixes to MapUnitPolys and ContactsAndFaults, it also only expects to find one pair of those. We could review that flexibility or change the tool as you suggest, but in your case, if you only selected the bedrock feature classes to be checked for valid topology, what about the surficial features? 

Do you (AKDGGS) use a topology to manage those four sets of features in two feature classes?
